src/main/java/com/icthh/xm/tmf/ms/resource/config/audit/AuditEventConverter.java

```diff
@@ -1,16 +1,12 @@
 package com.icthh.xm.tmf.ms.resource.config.audit;
 
 import com.icthh.xm.tmf.ms.resource.domain.PersistentAuditEvent;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
 import org.springframework.boot.actuate.audit.AuditEvent;
 import org.springframework.security.web.authentication.WebAuthenticationDetails;
 import org.springframework.stereotype.Component;
 
+import java.util.*;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version